packages/astro/components/ViewTransitions.astro

```diff
@@ -7,7 +7,20 @@ export interface Props {
 
 const { fallback = 'animate' } = Astro.props as Props;
```

Comment:
Unrelated to this PR, but is the `as Props` needed?

Reply:
It was when I worked on vscode 😡